platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/model/internal/JvmAnchor.java

```diff
@@ -33,16 +30,10 @@ public class JvmAnchor implements Startable, Stoppable {
 
     /**
      * Constructor.
-     *
-     * @param threadManager the thread manager
      */
-    public JvmAnchor(@NonNull final ThreadManager threadManager) {
-        thread = new ThreadConfiguration(threadManager)
-                .setComponent("platform")
-                .setThreadName("jvm-anchor")
-                .setRunnable(this::run)
-                .setDaemon(false)
-                .build();
+    public JvmAnchor() {
+        thread = new Thread(this::run, "<jvm-anchor>");
+        thread.setDaemon(false); // important
```

Comment:
Actually, looks like we don't expect to ever have more than one anchor instance. If so, and considering the contract of StandardWiringModel#preventJvmExit(), it might be worth making this a singleton to prevent inadvertently having multiple anchors?

Reply:
In a production environment we won't have more than one anchor. In TURTLE mode we may have multiple anchors, although having more than one is not strictly necessary.

The reason why I am extremely averse to using a true singleton model is that I believe static stateful variables to be inherently "evil". We are currently in the middle of converting all static singleton objects used by the platform into objects that are scoped to the platform, not to the JVM. To me, it's worth a little waste in TURTLE style tests to avoid having to use a static variable.